src/server/controllers/sendChangePasswordEmail.ts

```diff
@@ -162,10 +162,12 @@ export const sendEmailInOkta = async (
           // we still want to send a reset password email for these users
           // so we have to check for the correct error response
           // set an placeholder unknown password and then send the email
+          // E0000006 - Access denied exception
+          // E0000017 - Reset password failed exception
           if (
             isOktaError(error) &&
             error.status === 403 &&
-            error.code === 'E0000006'
+            (error.code === 'E0000006' || error.code === 'E0000017')
```

Comment:
Will revert this as TS complains `Argument of type 'string | undefined' is not assignable to parameter of type 'string'. Type 'undefined' is not assignable to type 'string'.` about `error.code` and it seems worse to also be checking that `.code` is defined

Reply:
rather than reverting this, it's best to just rewrite the git history and remove the commit completely using git rebase

```
git rebase -i origin/main
```

and mark the commit as `drop` or `d`